crates/std_detect/src/detect/arch/riscv.rs

```diff
@@ -146,7 +135,7 @@ features! {
     /// "Zfh" Standard Extension for 16-Bit Half-Precision Floating-Point
     @FEATURE: #[unstable(feature = "stdarch_riscv_feature_detection", issue = "111192")] zfhmin: "zfhmin";
     /// "Zfhmin" Standard Extension for Minimal Half-Precision Floating-Point Support
-    @FEATURE: #[unstable(feature = "stdarch_riscv_feature_detection", issue = "111192")] b: "b";
+    @FEATURE: #[stable(feature = "riscv_ratified", since = "1.76.0")] b: "b";
```

Comment:
"b" as an extension doesn't actually exist. Instead we should only expose the underlying zba/zbb/zbs extension.

Reply:
The "zb* does exist in the target features. Should we expose that?

Reply:
I don't see "zb" in `llc -march=riscv64 -mattr=help` or in https://github.com/rust-lang/rust/pull/116485.

Reply:
Okay, I removed the `zb` and `b` features.